include/gz/gui/qml/Main.qml

```diff
@@ -101,11 +101,11 @@ ApplicationWindow
   // C++ signals to QML slots
   Connections {
     target: MainWindow
-    onNotify: {
+    function onNotify(_message) {
```

Comment:
Unfortunately, according to this [commit message](https://github.com/qt/qtdeclarative/commit/a2eef6b511988b2435c4e39b6b5551e857ce7775), this syntax is not available for `Qt 5.12` which is the version for focal. A suggestion to look into is disable the warning or figure out how to call these functions from C++ code

Reply:
If we aren't planning on keeping `focal` support for `harmonic`, I can retarget to main, at least.

Reply:
I don't think there have been discussions or decisions on this yet

Reply:
I think the plan is to support `focal` and `jammy` for `harmonic`. https://github.com/gazebosim/docs/pull/324/files#diff-a056469a43d02c042f0ca8d049d90ae664ceffca6a9803b6520617b3fdbdd61e

Reply:
From the commit message @jennuine linked, perhaps we can disable the warning using  `QLoggingCategory::setFilterRules("qt.qml.connections=false");`?

Reply:
Alternative PR https://github.com/gazebosim/gz-gui/pull/534

Reply:
> From the commit message @jennuine linked, perhaps we can disable the warning using `QLoggingCategory::setFilterRules("qt.qml.connections=false");`?

Apologies, I missed this message but thanks for taking care of it :bowing_woman: